StreamChatClient_v3/WebSocketClient/Events/EventType.swift

```diff
@@ -90,7 +108,20 @@ enum EventType: String, Decodable {
         case .reactionUpdated: return try ReactionUpdatedEvent(from: response)
         case .reactionDeleted: return try ReactionDeletedEvent(from: response)
             
+        case .notificationMessageNew: return try NotificationMessageNewEvent(from: response)
+            
+        case .notificationMarkRead:
+            return response.channel == nil
+                ? try NotificationMarkAllReadEvent(from: response)
+                : try NotificationMarkReadEvent(from: response)
```

Comment:
It's not your fault but this is 🤮

Reply:
What's wrong with this?

Reply:
The fact that the presence of `channelId` differentiates between two different events. As I said, not your fault :) Just the design of these events is not the best...